docs/src/components/DocSearch.css

```diff
@@ -170,10 +170,10 @@
 }
 
 .algolia-autocomplete .ds-dropdown-menu {
+  composes: shadow from "./Borders.css";
```

Comment:
@themindfuldev could we remove this composes? This is CSS only for the docs themselves, not to be exported so we shouldn't depend on other Gestalt CSS

Reply:
Sure